packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -5768,6 +5778,11 @@ abstract class RenderObjectElement extends Element {
 
   @override
   void unmount() {
```

Comment:
Update the docs on unmount to indicate that that's also the place where resources may get released?

Reply:
Done